### PR TITLE
Adding Class Option back to Image Component

### PR DESCRIPTION
**Pull request by mayaraman19**
### Stories/Links:

Vaguely related to [DOP-4342](https://jira.mongodb.org/browse/DOP-4342)

[DOP-4321](https://github.com/mongodb/snooty/commit/862eb93807c02a3a880e1a420a4892c525d932a9#diff-ffd2b22195ccbbae4128ae0f9aa63e3b223dec62b101275024fd4d21509f58e6) resulted in, for some reason or another, the `:class:` option for directives not being passed to the `` tag anymore. This is a quick fix for that, as it was messing with the header on the new homepage.

### Current Behavior:

[Current behavior on draft homepage](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/DOP-4300/landing/maya/main/index.html)

### Staging Links:

[Updated behavior](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/DOP-4300/landing/maya/DOP-4342/)

Please ignore the "lg-ArrowRight" stuff going on!

### README updates

- [ ] This PR introduces changes that should be reflected in the README, and I have made those updates.
- [x] This PR does not introduce changes that should be reflected in the README

### CODE REVIEW

**Review by mmeigs — COMMENTED** [review]
Ahhh, I think I see what @seungpark might have done here that regressed. It's confusing, so I want others to double-check me.

Previously this class was grabbed like so: 

`const { options: { class: directiveClass } = {} } = nodeData;`

I believe this renames "class" to be `directiveClass` while saving it to a constant. When switching things over he is accessing `directiveClass` on `nodeData.options` and I believe that just doesn't ever exist. 

**Tldr;** I _believe_ we can completely erase all mentions of `directiveClass` in this file! But someone double-check me. Thanks!

**Comment by mayaraman19** [comment]
I tried to make it fewer changes, keeping `directiveClass` but accessing it in a different way! Will add people to double check :)

**Review by rayangler — APPROVED** [review]
Good catch! This lgtm. 

I think it might be good to add a class option to one of the test cases' mock data for the `Figure` test to ensure a regression like this doesn't happen again, but this is not a blocker
